tests/test_gtf_annotation.py

```diff
@@ -0,0 +1,573 @@
+"""
+Unit tests for GTF annotation functions.
+
+Tests the helper functions _empty_annot() and _get_strand_and_codon_pos()
+without requiring full SVAR objects or files.
+"""
+
+from __future__ import annotations
+
+import polars as pl
+import pyranges as pr
+import pytest
+from pytest_cases import parametrize_with_cases, case
+
+from genoray._svar import _get_strand_and_codon_pos, _empty_annot
+from genoray._utils import ContigNormalizer
+
+
+# Fixtures
+
+
+@pytest.fixture
+def basic_contig_normalizer():
+    """Standard human chromosome normalizer."""
+    return ContigNormalizer(["chr1", "chr2"])
+
+
+# Helper Functions for Expected Values
+
+
+def expected_codon_pos_plus(pos0: int, cds_start0: int, frame: int) -> int:
+    """Positive strand: (relative position + (3 - frame) % 3) % 3"""
+    return (pos0 - cds_start0 + (3 - frame) % 3) % 3
```

Comment:
```suggestion
    return (pos0 - cds_start0 - frame) % 3
```
wolfram alpha says these expressions are equivalent, but this is less ops/easier to read

Reply:
sounds good